src/config/middleware.js

```diff
@@ -15,10 +15,14 @@ const checkSessionAndRedirect = (req, res, next) => {
     };
   }
 
-  // If the user doesn't have a session and is not on the homepage, redirect to Journey Guard Page
+  // If the user doesn't have a session and is not on the homepage, redirect to Journey Guard Page but if entering on / take them to welcome page
```

Comment:
I don't understand how we manage this scenario: "if user doesn't has any session and he is on the homepage"?
Functionally, if the visitor comes for the first time and load "/", what is the page he is seeing?

Reply:
The journey guard page

Reply:
oh wait I dont think my new code was pushed